### ISSUE TITLE
Openings action "Notify me when..."

### ISSUE THREAD

**Issue by jodator** [Joystream/pioneer#414]
Follow-up of #289.

*   Check the requirements & designs if needed

**Comment by MattAugustynowicz** [comment]
@dmtrjsg I do not think we need to shorten the group name in any way, and everything will fit on two lines. This is an edge case, and most titles will be shorter. I posted an example with 55 characters and corrected text (thanks for the idea! 🙏 )

Just leave 24px between the text bottom line and the edge of this box, and it will be great 🌟 
[👉 Figma link](https://www.figma.com/file/GlgN8uBRtvtMJtiOsdtDF7/Pioneer-Design?node-id=8178%3A225508)

<img width="681" alt="image" src="https://user-images.githubusercontent.com/95480823/158638823-afbe17c3-6eee-4385-af71-a60bd110d01b.png">
